musicname/ze_ffxiv_wanderers_palace_v4_5s.cfg

```diff
@@ -0,0 +1,17 @@
+"music"
+{
+    "answers1.mp3"                                                                          "FFXIV OST - Answers"
+    "answers2.mp3"                                                                          "FFXIV OST - Answers (Vocal)"
+    "answers3.mp3"                                                                          "FFXIV OST - Answers"
+    "bre@th--less.mp3"                                                                      "SawanoHiroyuki[nZk] - Bre@th//Less"
```

Comment:
Invalid music name. `//` will be parsed as a comment, resulting in incomplete keyvalue